lib/helpers/parsePayload.js

```diff
@@ -2,8 +2,9 @@
 
 const parsePayload = (payload) => {
 
+    const buf = Buffer.from(payload.content);
```

Comment:
There is no reason to rewrote `payload.content` since it is already of type `Buffer`